application/src/main/java/org/wecancodeit/serverside/repository/CommentRepo.java

```diff
@@ -1,4 +1,9 @@
 package org.wecancodeit.serverside.repository;
+import org.springframework.data.repository.CrudRepository;
+import org.wecancodeit.serverside.model.Comment;
 
-public interface CommentRepo {
+import java.util.Optional;
+
+public interface CommentRepo extends CrudRepository <Comment, String> {
+    Optional<Comment> findComment(String commentName);
```

Comment:
I can't run the code at the moment to check, but I believe this needs to be findByCommentName to work